docs/azure_arc_jumpstart/azure_arc_k8s/aks_iot_edge/aks_iot_edge_terraform/_index.md

```diff
@@ -0,0 +1,232 @@
+
+* Export the environment variables needed for the Terraform plan.
+
+    ```shell
+    export TF_VAR_client_id=<Your Azure service principal App ID>
+    export TF_VAR_client_secret=<Your Azure service principal App Password>
+    ```
+
+    > **Note: If you are running in a PowerShell environment, to set the Terraform environment variables, use the _Set-Item -Path env:_ prefix (see example below)**
+
+    ```powershell
+    Set-Item -Path env:TF_VAR_client_id
+    ```
+
+* Run the ```terraform init``` command which will download the Terraform AzureRM provider.
+
+    ![Screenshot showing terraform init being run](./01.png)
+
+* Run the ```terraform apply --auto-approve``` command and wait for the plan to finish.
+
+    Once the Terraform deployment is completed, a new Resource Group and all services (Vnet, Subnets, VMs, IoT Hub, EventHub, AKS Cluster) are created.
+
+    ![Screenshot showing terraform plan completing](./02.png)
+
+    ![Screenshot showing Azure Portal with AKS resource](./03.png)
+
+    ![Screenshot showing Azure Portal with AKS resource](./04.png)
+
+
+* In this scenario we will use a VM as a "simulation" of an IoT Edge device. To do this, we must register a new Edge device in our IoT Hub that we will later configure. 
+
+* In order to keep your local environment clean and untouched, we will use [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview) (located in the top-right corner in the Azure portal) to run the next commands:
+
+    ![Screenshot showing how to access Cloud Shell in Visual Studio Code](./05.png)
+
+* Create IoT Edge Device
+ ```shell
+az iot hub device-identity create --device-id "EdgeDeviceSim" --edge-enabled --hub-name k8sedgejumpstart
+ ```
+
+> **Note: We will obtain the connection string of the new IoT Edge device to be able to make the link**
+```shell
+az iot hub device-identity connection-string show --device-id "EdgeDeviceSim" --hub-name k8sedgejumpstart
+  ```
+
+  ![Screenshot showing how to access Cloud Shell in Visual Studio Code](./06.png)
+
+  ![Screenshot showing how to access Cloud Shell in Visual Studio Code](./07.png)
+
+* Now we must modify the file /etc/aziot/config.toml that is in the VM that we have deployed, for this we must connect by SSH with our credentials.
+Once inside the VM, we will modify the /etc/aziot/config.toml file including the following lines at the end of the file with the value of the connection string that we have obtained in the previous step: 
+
+```shell
+# Manual provisioning with connection string
+[provisioning]
+source = "manual"
+connection_string = "<ADD DEVICE CONNECTION STRING HERE>"
+ ```
+
+* We save the file and execute the following command to synchronize the configuration: 
```

Comment:
```suggestion
* We save the file and execute the following command to synchronize the configuration: 
```

The lingo here is a bit confusing and unclear. Try to think about how to write this as a guide and not like a face-to-face conversation.